quadfeather/tiler.py

```diff
@@ -82,6 +82,32 @@ def flatten_manifest(mani: TileManifest) -> List[Dict]:
     return d
 
 
+def rebatch(
+    input: Iterator[pa.RecordBatch], size: float, max_rows: float = float("inf")
```

Comment:
nit: size is float?

Reply:
IIRC python gets upset if you use scientific notation on an int type, but I want to be able to write size 4.5e6 or whatever.